sale_merge_draft_invoice/README.rst

```diff
@@ -0,0 +1,134 @@
+them.
+
+This feature is activated from Sales / Settings and assigned to the company
+settings.
+
+Users assigned to the group 'Change sale to invoice merge proposal'
+have the possibility to activate or deactivate this option at the moment of
+invoicing.
+
+If you also need to group on fields other than partner and currency, the OCA
+module sale_invoice_group_method allows you to specify alternative grouping
+methods and is fully compatible with this module.
+
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+#. Go to 'Sales / Configuration / Settings'
+#. In 'Quotations & Sales' go to 'Invoices' and activate the option 'Merge
+   new invoices with existing draft ones'.
+#. Go to 'Settings / Users / Groups' and assign to the group 'Change sale
+   to invoice merge proposal' all the users that can have the possibility to
+   activate/deactivate the option of merging the new invoice with existing
+   draft ones.
+
+
+Usage
+=====
+
+Create an Invoice from a single Sale Order
+------------------------------------------
+
+#. Go to 'Sales / Sales / Sales Orders'.
+#. Select or click in a sale order.
+#. Go to 'Action' and click 'Invoice Order' or press the button 'Invoice
+   Order' if you are inside a sale order. The result, in case there are draft
+   invoices that correspond to this one, is an existing draft invoice with
+   the new lines added.
+   In the wizard, if the user belongs to the group 'Change sale to invoice merge
+   proposal', the option to deactivate the 'Merge with draft invoices' is
+   given. In this case, the result is a new draft invoice is created
+   following the usual behaviour.
+
+Create Invoices from a group of Sales Orders
+--------------------------------------------
+
+#. Go to 'Sales / Sales / Sales Orders'.
+#. Select all the sales orders with status 'To invoice'.
+#. Go to 'Action' and click 'Invoice Order'. As a result, new draft invoices
+   will be merged with existing ones when corresponding.
+   In the wizard, if the user belongs to the group 'Change sale to invoice merge
+   proposal', the option to deactivate the 'Merge with draft invoices' is
+   given. In this case, the result is a new draft invoice for every sale
+   order following the usual behaviour.
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/sale-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/sale-workflow/issues/new?body=module:%20sale_merge_draft_invoice%0Aversion:%2012-sale_merge_draft_invoice%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
```

Comment:
Hi,
'~' causes error on install
docutils.utils.SystemMessage: <string>:105: (SEVERE/4) Title level inconsistent:

Authors
~~~~~~~
Kind regards
Jim

Reply:
Yes, I know , That's way `Mr.Travis` is Red